README.md

```diff
@@ -1,197 +1,152 @@
+publishing documentation for SailfishOS:Chum.
 
-The latter is at an early stage of development and has the most
-basic functionality. So be prepared to use the command line anyway:wink:
+For the etymological origin and meanings of the word "chum", see
+[en.wikipedia.org:Chumming](https://en.wikipedia.org/wiki/Chumming)
+and [en.wiktionary.org:chum](https://en.wiktionary.org/wiki/chum).
 
-### Where to get the packages
 
-You can download the required RPM files directly from
-[OBS](https://build.sailfishos.org/project/show/sailfishos:chum)
-or use this helper page [chumrpm.netlify.app](https://chumrpm.netlify.app/).
+## How to use the SailfishOS:Chum repository: User's guide
 
-### Command line tools
+There are two different ways of using the SailfishOS:Chum repository:
+- with the [SailfishOS:Chum GUI application](https://github.com/sailfishos-chum/sailfishos-chum-gui)
+- with the usual command line tools for package management, `pkcon` or `zypper`
 
-To use the repositories on your device, users would have to
-- install package `sailfishos-chum`;
-- refresh software cache on device (`devel-su zypper ref`);
-- install the software (`devel-su zypper in ...`)
+### How to install the SailfishOS:Chum GUI-client app
 
-As all software is compiled with the vendor set to `chum`, it is easy
-to find out which packages are installed from this repository by running
+The client app (GUI) for SailfishOS:Chum is available for easy installation at [chumrpm.netlify.app](https://chumrpm.netlify.app/) and its individual RPMs are also provided at [the SailfishOS:Chum repository](https://build.sailfishos.org/package/show/sailfishos:chum/sailfishos-chum-gui).
 
-```sh
-rpm -qa --queryformat '%{vendor}:%{name}\n' | grep chum
-```
+### How to deploy the configuration for command line tools
 
+For using the SailfishOS:Chum repository per command line tools, a `sailfishos-chum` helper RPM is available for easy installation also at [chumrpm.netlify.app](https://chumrpm.netlify.app/) (and additionally at the [SailfishOS:Chum repository](https://build.sailfishos.org/package/show/sailfishos:chum/sailfishos-chum) proper), which solely provides an appropriate local repository configuration for utilising the SailfishOS:Chum repository.<br />
+Note that installing the SailfishOS:Chum GUI-client deploys the same local repository configuration, so your device is already set for using the SailfishOS:Chum repository with the usual command line tools for package management, then.
 
-## How to use these repositories: developers
+To utilise the SailfishOS:Chum repository per command line tools on your device, you have to …
+1. download either of the aforementioned packages `sailfishos-chum` or `sailfishos-chum-gui`.
+2. install it per file manager or as root user per `pkcon install-local sailfishos-chum[-gui]` or `zypper in sailfishos-chum[-gui]`.
+3. refresh the software cache on your device as root user per `pkcon refresh` or `zypper ref`.
```

Comment:
per -> using

Reply:
Ack.

Reply:
… decided for "by executing" the second occurrence.